fpdf/fpdf.py

```diff
@@ -1974,7 +1974,15 @@ def _out(self, s):
         if(self.state == 2):
             self.pages[self.page]["content"] += (s + "\n")
         else:
-            self.buffer += (s + "\n")
+            if PY3K:
+                if not isinstance(s, bytes):
```

Comment:
Are these checks really needed? According to the logic above, it seems in PY3K _s_ will already have been decoded from bytes, and in Python 2 _s_ will always have been converted to str.